app/data/census_household.json

```diff
@@ -960,7 +981,8 @@
                 {
                     "repeat": {
                         "type": "answer_count",
-                        "answer_id": "first-name"
+                        "answer_id": "first-name",
+                        "hyperlink_name_format":["first-name", "middle-names", "last-name"]
```

Comment:
I don't think it should contain middle-names; let's check with Louise in the morning.